frontend/.envexample

```diff
@@ -10,4 +10,4 @@ BACKEND_URL=http://localhost:5000
 
 # sentry 
 SENTRY_DSN=https://PUBLICKEY@sentry.io/PROJECTID
-SENTRY_RELEASE="optional, overrides git hash"
+SENTRY_RELEASE="optional, provides sentry logging with release info"
```

Comment:
Should we comment out by default, like backend/.env.example? We probably don't want this being configured with this in dev.

Reply:
Yes, I'll do this.